trade-finance-manager-ui/templates/case/amendments/amendment-task.njk

```diff
@@ -4,6 +4,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle -%}Facility {{ amendment.ukefFacilityId }} - Task {{ task.title }}{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.